src/settings/RequestPolicy/RequestPolicyForm.test.js

```diff
@@ -0,0 +1,309 @@
+import React from 'react';
+import {
+  render,
+  screen,
+  fireEvent,
+} from '@testing-library/react';
+
+import '../../../test/jest/__mock__';
+
+import {
+  AccordionSet,
+  ExpandAllButton,
+  Pane,
+} from '@folio/stripes/components';
+
+import RequestPolicyForm from './RequestPolicyForm';
+import { GeneralSection } from './components';
+import {
+  CancelButton,
+  FooterPane,
+} from '../components';
+import RequestPolicy from '../Models/RequestPolicy';
+
+jest.mock('@folio/stripes/final-form', () => jest.fn(() => (component) => component));
+jest.mock('./components', () => ({
+  GeneralSection: jest.fn(() => null),
+}));
+jest.mock('../components', () => ({
+  CancelButton: jest.fn(() => null),
+  FooterPane: jest.fn(() => null),
+}));
+
+Pane.mockImplementation(({
+  children,
+  firstMenu,
+  footer,
+}) => (
+  <div>
+    {firstMenu}
+    {children}
+    {footer}
+  </div>
+));
```

Comment:
It is redundant. We already have the same mock here - https://github.com/folio-org/ui-circulation/blob/e6f072c3e32ac577021692c3a4d6cefc51356127/test/jest/__mock__/stripesComponents.mock.js#L88

Reply:
You are right! Thanks!